src/resqml2_0_1/FluidBoundaryFeature.cpp

```diff
@@ -17,16 +17,18 @@ specific language governing permissions and limitations
 under the License.
 -----------------------------------------------------------------------*/
 #include "resqml2_0_1/FluidBoundaryFeature.h"
+#include "tools/Misc.h"
 
 #include <stdexcept>
 
 using namespace std;
 using namespace RESQML2_0_1_NS;
 using namespace gsoap_resqml2_0_1;
+using namespace epc;
 
 const char* FluidBoundaryFeature::XML_TAG = "FluidBoundaryFeature";
 
-FluidBoundaryFeature::FluidBoundaryFeature(soap* soapContext, const string & guid, const string & title, const resqml2__FluidContact & fluidContact)
+FluidBoundaryFeature::FluidBoundaryFeature(soap* soapContext, const string & guid, const string & title, const resqml2__FluidContact & fluidContact) : rockFluidUnitFeature(nullptr)
```

Comment:
rockFluidUnitFeature should be a vector (there can be several units using this boundary as a top or bottom).
Ideally, this vector should be in BoundaryFeature superclass according to RESQML datamodel